ruskeyverbs/templates/ruskeyverbs/signup.html

```diff
@@ -0,0 +1,53 @@
+{% extends "ruskeyverbs/message.html" %}
+{% block message_content %}
+  <h2>Sign up</h2>
+  <div id='istaken'>
+    <form method="post">
+      {% csrf_token %}
+
+        {% for field in form %}
+        <p>
+          {{ field.label_tag }}<br>
+
+          {{ field }}
+          {% if field == form.username %}
+          <span id='usermessage' style="font-weight:bold">[[warningmessage]]</span>
+          {% endif %}
+          {% if field.help_text %}
+            <small style="display: none">{{ field.help_text }}</small>
+          {% endif %}
+          {% for error in field.errors %}
+            <p style="color: red">{{ error }}</p>
+          {% endfor %}
+        </p>
+        {% endfor %}
+      <button type="submit">Sign up</button>
+
+    </form>
+  </div>
+  <script>
+  var vm = new Vue({
+    delimiters: ['[[', ']]'],
+    el: '#istaken',
+    data: {
+      name: '',
+      warningmessage: ''
+    },
+    methods: {
+      TestForUser: function() {
+        axios.get("/userapi/"+this.name)
+        .then(function(response) {
+          if (response.data['username'] == vm.name) {
+              vm.$data['warningmessage'] = 'username already in use!';
+              document.getElementById('usermessage').style.color = 'red';
```

Comment:
A slightly smoother Vue way to do this would be to bind the style of the element to a Vue model...